dev/devicelab/bin/tasks/flavors_test_ios.dart

```diff
@@ -25,6 +25,7 @@ Future<void> main() async {
       final List<TaskResult> testResults = <TaskResult>[
         await _testInstallDebugPaidFlavor(projectDir),
         await _testInstallBogusFlavor(),
+        await _testFlavorsWorkWhenBuiltWithXcodebuild(projectDir),
```

Comment:
Similarly to my previous comment, would `_testFlavorsWhenBuildStartsWithXcode` be descriptive of the fact that the build does not start with the `flutter` CLI?

Reply:
I agree, thanks!